src/main/kotlin/no/nav/familie/ef/iverksett/tilbakekreving/TilbakekrevingListener.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.familie.ef.iverksett.tilbakekreving
+
+import no.nav.familie.ef.iverksett.felles.FamilieIntegrasjonerClient
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.kontrakter.felles.arbeidsfordeling.Enhet
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.kontrakter.felles.tilbakekreving.HentFagsystemsbehandlingRequest
+import no.nav.familie.log.mdc.MDCConstants
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+import org.springframework.kafka.annotation.KafkaListener
+import org.springframework.kafka.listener.ConsumerSeekAware
+import org.springframework.stereotype.Component
+import java.util.UUID
+
+@Component
+class TilbakekrevingListener(
+        private val iverksettingRepository: IverksettingRepository,
+        private val familieIntegrasjonerClient: FamilieIntegrasjonerClient,
+        private val tilbakekrevingProducer: TilbakekrevingProducer
+) : ConsumerSeekAware {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    @KafkaListener(id = "familie-ef-iverksett",
+                   topics = ["teamfamilie.privat-tbk-hentfagsystemsbehandling-request-topic"],
+                   containerFactory = "concurrentTilbakekrevingListenerContainerFactory")
+    fun listen(consumerRecord: ConsumerRecord<String, String>) {
+        secureLogger.info("HentFagsystemsbehandlingRequest er mottatt: $consumerRecord")
```

Comment:
Trenger vi å logge allt vi mottar på kafka?

Reply:
Jeg er usikker. Trenger vi det ?

Reply:
For meg trenger vi ikke det, er ofte kun interessant de ganger det feiler?

Reply:
Ja, fjerner denne